nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ConfigureProxy.java

```diff
@@ -84,51 +84,21 @@ public void execute() throws BuildException {
     static URLConnection openConnection(Task task, final URL url, URI[] connectedVia) throws IOException {
         final URLConnection[] conn = { null };
         final List<Exception> errs = new CopyOnWriteArrayList<>();
+        final StringBuilder msgs = new StringBuilder();
```

Comment:
Isn't `msgs` operated from different threads ? Noticed only now :(

Reply:
E.g. it should be `StringBuffer`, not builder. Right?

Reply:
Done in 40584ecfeb52cfd7f8f5bc619ad97ad968303819 - if approved, I also merge it to `master`.

Btw. the code was ready for being used from multiple threads - see all the `msgs.append("x' + a + " y " + b + " z" + c)` concatentations that are done in single thread and only then placed into the `msgs` object. Why did I not used `StringBuffer` then? I know I wanted...